src/Program.cs

```diff
@@ -28,6 +28,7 @@
 using CommandLine;
 using FluentValidation;
 using FluentValidation.Results;
+using PhotoCli.Models.ReverseGeocode;
```

Comment:
Shall we make these using global like the others on top and remove on other files? As we don't have much classes and conflicts this global usages make class files more light.